<ui version="4.0" >
 <class>CGexFtpPage_widget_base</class>
 <widget class="QWidget" name="CGexFtpPage_widget_base" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FTP page</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="tabStatus" >
      <attribute name="title" >
       <string>Status</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QStackedWidget" name="stackedWidget" >
         <widget class="QWidget" name="pageIdle" >
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <spacer name="spacer16" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>10</width>
               <height>91</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="_2" >
             <item>
              <spacer name="spacer19" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>186</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="textLabel1" >
               <property name="text" >
                <string>&lt;font color="#0000ff">&lt;b>&lt;i>&lt;h1>&lt;font face="Times New Roman">Idle...&lt;/font>&lt;/h1>&lt;/i>&lt;/b>&lt;/font></string>
               </property>
               <property name="wordWrap" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="spacer18" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>241</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="spacer17" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Expanding</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>10</width>
               <height>147</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
